agent/consul/multilimiter/multilimiter_test.go

```diff
@@ -279,8 +298,29 @@ func TestRateLimiterUpdateConfig(t *testing.T) {
 		limiter := l.(*Limiter)
 		require.True(t, c1.isApplied(limiter.limiter))
 	})
+
+	t.Run("Allow an IP with prefix and check prefix defaultConfig is applied to new keys under that prefix, delete config and check default applied", func(t *testing.T) {
+		c := Config{ReconcileCheckLimit: 100 * time.Second, ReconcileCheckInterval: 10 * time.Millisecond}
+		m := NewMultiLimiter(c)
+		require.Equal(t, *m.defaultConfig.Load(), c)
+		c1 := LimiterConfig{Rate: 3}
+		prefix := []byte("namespace.read")
+		m.UpdateConfig(c1, prefix)
+		ip := Key(prefix, []byte("127.0.0.1"))
+		m.Allow(ipLimited{key: ip})
+		storeLimiter(m)
+		l, ok := m.limiters.Load().Get(ip)
+		require.True(t, ok)
+		require.NotNil(t, l)
+		limiter := l.(*Limiter)
+		require.True(t, c1.isApplied(limiter.limiter))
+		m.DeleteConfig(prefix)
+		reconcile(m)
+		_, ok = m.limiters.Load().Get(ip)
+		require.False(t, ok)
+	})
```

Comment:
nit: would you mind adding separating out the blocks with newlines to make this easier to follow? 🙇🏻‍♂️